Lib/idlelib/editor.py

```diff
@@ -134,6 +134,10 @@ def __init__(self, flist=None, filename=None, key=None, root=None):
                         'main', 'EditorWindow', 'height', type='int'),
                 }
         self.text = text = MultiCallCreator(Text)(text_frame, **text_options)
+        # Store the current value of the insertofftime now so we can restore
+        # it if needed.
+        self.text._insertofftime = self.text['insertofftime']
```

Comment:
The attribute should be attached to self, not the tk text, as with other instance attributes.
```suggestion
        self.blink_off_time = self.text['insertofftime']
```
Corresponding change is suggested below.

The NMT tkinter 8.5 manual says that this is 300, but the tcl/tk 8.6 reference does not give a default, so grabbing the initial time seems safest for matching blinks on a particular system.

This set of new lines belongs further down the file, perhaps following 
`        self.set_notabs_indentwidth()`